geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/partitions/AttributeScheme.scala

```diff
@@ -101,7 +104,11 @@ object AttributeScheme {
         val binding = sft.getDescriptor(index).getType.getBinding
         require(AttributeIndexKey.encodable(binding),
           s"Invalid type binding '${binding.getName}' of attribute '$attribute'")
-        Some(AttributeScheme(attribute, index, binding))
+        val enums: Seq[String] = config.options.getOrElse(Config.EnumerationOpt, null) match {
```

Comment:
pedantic, but slightly cleaner might be `config.options.get(Config.EnumerationOpt).map(s.split(",")).getOrElse(Seq.empty)`